incomplete-cat-B-items.md

```diff
@@ -0,0 +1,116 @@
+This file has been auto-generated based on README.md. Do not edit it manually
+
+| Cat | Status   | Reference | Category                        | Description                                             | Comment
+| --- | -------- | --------- | ------------------------------- | ------------------------------------------------------- | -------
+|  B+ |          | 2.4.2     | theme                           | `titleLayout`                                           |
+|  B  |          | 1.2.6     | wayland-protocol                | input-method-unstable-v1                                |
+|  B  |          | 1.2.7     | wayland-protocol                | input-timestamps-unstable-v1                            |
+|  B  |          | 1.2.8     | wayland-protocol                | keyboard-shortcuts-inhibit-unstable-v1                  |
```

Comment:
This protocol might actually annoy users, apparently it is used by Chromium for example which prevents \*any\* compositor keybinds to work as long as it has keyboard focus. So if we decide to implement it we should likely combine it with at least a global config option to disable it completely or even integrate it with the policy framework so users can prevent applications like Chromium to take advantage of it.

Reply:
Interesting. I suggest we annotate the scope-document as such. It's a pretty clean way of keeping track of ours thoughts/positions on these things.

Reply:
Seems only to happen when using chromium in `--app` mode (possibly including Electron, not sure).
But any application being able to prevent compositor keybinds to work should likely be whitelisted in some way.

Some links regarding the chromium behavior:

- https://www.reddit.com/r/swaywm/comments/vuhr75/why_does_chromium_with_app_now_grab_complete/.compact
- https://www.reddit.com/r/swaywm/comments/vkgfza/chromium_v103_application_mode_app_and_weird/.compact
- https://www.reddit.com/r/swaywm/comments/vlvpws/chromeium_app_flag_spawns_windows_with_shortcut/.compact
- https://github.com/chromium/chromium/commit/ac0160a4ae8b8432a2041992415ceb1c1b0597e0